firmware/src/boards/cynthion_d11/fpga.c

```diff
@@ -11,25 +11,41 @@
 #include "apollo_board.h"
 #include "jtag.h"
 #include "fpga.h"
+#include "board_rev.h"
 
 
+#ifdef BOARD_HAS_FPGA_SYSCFG_PINS
+static bool board_has_fpga_initn(void)
+{
+#if (_BOARD_REVISION_DETECT_ == 1)
+	uint16_t revision = get_board_revision();
+    const uint8_t major = revision >> 8;
+    const uint8_t minor = revision & 0xFF;
+    return major == 1 && minor > 2;
```

Comment:
I don't like having to do all this every time we ever want to compare board revisions. I feel like either these four lines of code should be replaced with `return get_board_revision() > BOARD_REV_1_2;` or `board_rev.h` should provide a comparison function or two.